### ISSUE TITLE
[TASK] - [A2] Website Color Themes

### ISSUE THREAD

**Issue by Crynner** [fn5-310/movie-recommender#51]
**What is the task? Why is it needed?**
The primary theme of the website is darker-oriented, but for user preference and accessibility implications, it would be ideal to implement a selection of color palettes that would fit the website.

**What needs to be done?**
Somewhere on the page, there should be an option to change the theme of the website. This should implement the options for light, dark, and auto (i.e. dependent on system preference), having the dark theme as the default. Optionally, there may be other hue palettes adapted (e.g. green, blue, etc.).

The user should have the ability to change this at any time.

All palettes should be created as to fit within WCAG contrast guidelines level AA (ideally AAA).

**How can the implementation be tested?**
- Testing accessibility guidelines (namely [WCAG text contrast levels](https://webaim.org/resources/contrastchecker/))
- Testing color palette functionality
- Testing all elements still render as expected
